tree/dataframe/inc/ROOT/RDFHelpers.hxx

```diff
@@ -181,6 +184,110 @@ RNode AsRNode(NodeType node)
 // clang-format on
 void RunGraphs(std::vector<RResultHandle> handles);
 
+
+// clang-format off
+/// RDF progress helper.
+/// This class provides callback functions to RDataFrame, which record event statistics
+/// and print them to the terminal every second.
+/// ProgressHelper::operator()(unsigned int, T&) is thread safe, and can be used as callback in MT mode.
```

Comment:
Can you show the actual code to be used, given an `RDataFrame rdf`?

Reply:
Code not finalised yet (see TODOs), but yes!